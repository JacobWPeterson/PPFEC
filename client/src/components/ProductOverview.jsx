import React from 'react';
import Styles from './StylesComponent.jsx';

const ProductOverview = ({ details, styles }) => (
  <div>
    <div className="productInfo">
      <div>Ratings component will go here</div>
      <div>{details.category}</div>
      <div>{details.name}</div>
      <div>{details.default_price}</div>
    </div>
    <Styles styles={styles} />
    <div>Buttons and size selectors go here</div>
  </div>
);

export default ProductOverview;
