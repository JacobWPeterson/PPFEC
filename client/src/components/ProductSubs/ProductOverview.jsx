import React from 'react';
import styled from 'styled-components';

import Styles from './StylesComponent.jsx';
import Selectors from './Selectors.jsx';

const Wrapper = styled.div`
  flex_basis 40%;
  display: flex;
  margin-top: 5vh;
  flex-direction: column;
  padding-left: 10px;
`;

const Category = styled.p`
  margin-top: 5vh;
  font-weight: thin;
`;

const Name = styled.h1`
  line-height: 0;
  margin-bottom: 1vh;
  font-weight: bold;
`;

const Price = styled.p`
  margin-top: 5vh;
  margin-bottom: 1vh;
  font-weight: thin;
`;

const PurchaseLikeButtons = styled.div`
  width: 100%;
  display: flex;
  margin-top: 10px;
  flex-direction: row;
`;

const AddToCart = styled.button`
  flex-basis 80%;
  display: flex;
  flex-direction: row;
  flex-wrap: no-wrap;
  justify-content: space-between;
  align-items: center;
  font-weight: bold;
  width: 15vw;
  height: 5vh;
  padding: 0 1.25vw;
  background: none;
  border: 1px solid #424242;
  color: #424242;
`;

const Like = styled.button`
  flex-basis 20%;
  height: ${AddToCart.height};
  min-width: ${AddToCart.height};
  margin-left: 1vw;
  font-weight: bold;
  background: none;
  border: 1px solid #424242;
  color: #424242;
  &:hover { color: #80ccc4; };
`;

const ProductOverview = ({
  details, styles, skus, getStyleID,
}) => (
  <Wrapper>
    <div>Ratings component will go here</div>
    <div className="productInfo">
      <Category>{details.category.toUpperCase()}</Category>
      <Name>{details.name}</Name>
      <Price>{`$${details.default_price}`}</Price>
    </div>
    <Styles styles={styles} getStyleID={getStyleID} />
    <Selectors skus={skus} />
    <PurchaseLikeButtons>
      <AddToCart>
        <div>ADD TO BAG</div>
        <div>+</div>
      </AddToCart>
      <Like>â˜†</Like>
    </PurchaseLikeButtons>
  </Wrapper>
);

export default ProductOverview;
