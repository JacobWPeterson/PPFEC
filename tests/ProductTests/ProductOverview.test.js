/* eslint-disable no-undef */
/* eslint-disable react/jsx-props-no-spreading */
import React from 'react';
import { mount } from 'enzyme';

import ProductOverview from '../../client/src/components/ProductSubs/ProductOverview.jsx';

const props = {
  details: { category: 'Pants', name: 'Morning Joggers' },
  skus: {
    407566: { quantity: 8, size: 'XS' },
    407567: { quantity: 16, size: 'S' },
    407568: { quantity: 17, size: 'M' },
    407569: { quantity: 10, size: 'L' },
    407570: { quantity: 15, size: 'XL' },
    407571: { quantity: 6, size: 'XXL' },
  },
  styles: {
    product_id: '14036',
    results: [
      {
        style_id: 70550, name: 'Black', original_price: '40.00', sale_price: null, photos: [{ thumbnail_url: 'https://images.unsplash.com/photo-1552902865-b72c031ac5ea?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80' }],
      },
      {
        style_id: 70551, name: 'Grey', original_price: '40.00', sale_price: null, photos: [{ thumbnail_url: 'https://images.unsplash.com/photo-1562542082-519ebcdb43e6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80' }],
      },
      {
        style_id: 70552,
        name: 'Goldenrod',
        original_price: '40.00',
        sale_price: '35.00',
        photos: [{ thumbnail_url: 'https://images.unsplash.com/photo-1530073391204-7b34a1497281?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80' }],
      },
      {
        style_id: 70553, name: 'Maroon', original_price: '40.00', sale_price: '35.00', photos: [{ thumbnail_url: 'https://images.unsplash.com/photo-1519857609704-61e751edba25?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80' }],
      },
      {
        style_id: 70554, name: 'Chartreuse', original_price: '40.00', sale_price: '25.00', photos: [{ thumbnail_url: 'https://images.unsplash.com/photo-1530073391204-7b34a1497281?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80' }],
      },
      {
        style_id: 70555, name: 'White', original_price: '40.00', sale_price: null, photos: [{ thumbnail_url: 'https://images.unsplash.com/photo-1510390099355-23e690d8129d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80' }],
      }],
  },
};

describe('ProductOverview', () => {
  const ProductOverviewComponent = mount(<ProductOverview {...props} debug />);

  it('Should render correctly', () => {
    expect(ProductOverviewComponent).toMatchSnapshot();
  });
});
