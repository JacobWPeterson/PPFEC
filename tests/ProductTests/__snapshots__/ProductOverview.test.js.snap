// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductOverview Should render correctly 1`] = `
<ProductOverview
  debug={true}
  details={
    Object {
      "category": "Pants",
      "name": "Morning Joggers",
    }
  }
  skus={
    Object {
      "407566": Object {
        "quantity": 8,
        "size": "XS",
      },
      "407567": Object {
        "quantity": 16,
        "size": "S",
      },
      "407568": Object {
        "quantity": 17,
        "size": "M",
      },
      "407569": Object {
        "quantity": 10,
        "size": "L",
      },
      "407570": Object {
        "quantity": 15,
        "size": "XL",
      },
      "407571": Object {
        "quantity": 6,
        "size": "XXL",
      },
    }
  }
  styles={
    Object {
      "product_id": "14036",
      "results": Array [
        Object {
          "name": "Black",
          "original_price": "40.00",
          "photos": Array [
            Object {
              "thumbnail_url": "https://images.unsplash.com/photo-1552902865-b72c031ac5ea?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
            },
          ],
          "sale_price": null,
          "style_id": 70550,
        },
        Object {
          "name": "Grey",
          "original_price": "40.00",
          "photos": Array [
            Object {
              "thumbnail_url": "https://images.unsplash.com/photo-1562542082-519ebcdb43e6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
            },
          ],
          "sale_price": null,
          "style_id": 70551,
        },
        Object {
          "name": "Goldenrod",
          "original_price": "40.00",
          "photos": Array [
            Object {
              "thumbnail_url": "https://images.unsplash.com/photo-1530073391204-7b34a1497281?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
            },
          ],
          "sale_price": "35.00",
          "style_id": 70552,
        },
        Object {
          "name": "Maroon",
          "original_price": "40.00",
          "photos": Array [
            Object {
              "thumbnail_url": "https://images.unsplash.com/photo-1519857609704-61e751edba25?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
            },
          ],
          "sale_price": "35.00",
          "style_id": 70553,
        },
        Object {
          "name": "Chartreuse",
          "original_price": "40.00",
          "photos": Array [
            Object {
              "thumbnail_url": "https://images.unsplash.com/photo-1530073391204-7b34a1497281?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
            },
          ],
          "sale_price": "25.00",
          "style_id": 70554,
        },
        Object {
          "name": "White",
          "original_price": "40.00",
          "photos": Array [
            Object {
              "thumbnail_url": "https://images.unsplash.com/photo-1510390099355-23e690d8129d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
            },
          ],
          "sale_price": null,
          "style_id": 70555,
        },
      ],
    }
  }
>
  <ProductOverview__Wrapper>
    <div
      className="ProductOverview__Wrapper-sc-1yvw4zm-0 defTdM"
    >
      <ProductOverview__RatingWrapper>
        <div
          className="ProductOverview__RatingWrapper-sc-1yvw4zm-1 gPkqin"
        >
          <ProductOverview__Rating>
            <div
              className="ProductOverview__Rating-sc-1yvw4zm-2 gbslIc"
            >
              ★★★★☆
            </div>
          </ProductOverview__Rating>
          <ProductOverview__RatingsAnchor
            href="#RATINGS"
          >
            <a
              className="ProductOverview__RatingsAnchor-sc-1yvw4zm-3 fDtPuq"
              href="#RATINGS"
            >
              Read all reviews
            </a>
          </ProductOverview__RatingsAnchor>
        </div>
      </ProductOverview__RatingWrapper>
      <ProductOverview__Category>
        <p
          className="ProductOverview__Category-sc-1yvw4zm-4 KvVVf"
        >
          PANTS
        </p>
      </ProductOverview__Category>
      <ProductOverview__Name>
        <h1
          className="ProductOverview__Name-sc-1yvw4zm-5 WXLyo"
        >
          Morning Joggers
        </h1>
      </ProductOverview__Name>
      <Styles
        styles={
          Object {
            "product_id": "14036",
            "results": Array [
              Object {
                "name": "Black",
                "original_price": "40.00",
                "photos": Array [
                  Object {
                    "thumbnail_url": "https://images.unsplash.com/photo-1552902865-b72c031ac5ea?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
                  },
                ],
                "sale_price": null,
                "style_id": 70550,
              },
              Object {
                "name": "Grey",
                "original_price": "40.00",
                "photos": Array [
                  Object {
                    "thumbnail_url": "https://images.unsplash.com/photo-1562542082-519ebcdb43e6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
                  },
                ],
                "sale_price": null,
                "style_id": 70551,
              },
              Object {
                "name": "Goldenrod",
                "original_price": "40.00",
                "photos": Array [
                  Object {
                    "thumbnail_url": "https://images.unsplash.com/photo-1530073391204-7b34a1497281?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
                  },
                ],
                "sale_price": "35.00",
                "style_id": 70552,
              },
              Object {
                "name": "Maroon",
                "original_price": "40.00",
                "photos": Array [
                  Object {
                    "thumbnail_url": "https://images.unsplash.com/photo-1519857609704-61e751edba25?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
                  },
                ],
                "sale_price": "35.00",
                "style_id": 70553,
              },
              Object {
                "name": "Chartreuse",
                "original_price": "40.00",
                "photos": Array [
                  Object {
                    "thumbnail_url": "https://images.unsplash.com/photo-1530073391204-7b34a1497281?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
                  },
                ],
                "sale_price": "25.00",
                "style_id": 70554,
              },
              Object {
                "name": "White",
                "original_price": "40.00",
                "photos": Array [
                  Object {
                    "thumbnail_url": "https://images.unsplash.com/photo-1510390099355-23e690d8129d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
                  },
                ],
                "sale_price": null,
                "style_id": 70555,
              },
            ],
          }
        }
      >
        <StylesComponent__StyleWrapper>
          <div
            className="StylesComponent__StyleWrapper-ajk3ts-0 bQXBFa"
          >
            <Price
              price="40.00"
              sale={null}
            >
              <Price__NoSale>
                <p
                  className="Price__NoSale-eho440-0 ituvCN"
                >
                  $40
                </p>
              </Price__NoSale>
            </Price>
            <StylesComponent__TextWrapper>
              <div
                className="StylesComponent__TextWrapper-ajk3ts-1 kDJaHh"
              >
                <StylesComponent__Style>
                  <p
                    className="StylesComponent__Style-ajk3ts-2 glNYdS"
                  >
                    STYLE &gt;
                  </p>
                </StylesComponent__Style>
                <StylesComponent__SelectedStyle>
                  <p
                    className="StylesComponent__SelectedStyle-ajk3ts-3 RbxlX"
                  >
                    BLACK
                  </p>
                </StylesComponent__SelectedStyle>
              </div>
            </StylesComponent__TextWrapper>
            <StylesComponent__Thumbs>
              <div
                className="StylesComponent__Thumbs-ajk3ts-4 gBVoCt"
              >
                <StylesComponent__SelectedImage
                  alt="Black"
                  id={70550}
                  key="70550"
                  onClick={[Function]}
                  src="https://images.unsplash.com/photo-1552902865-b72c031ac5ea?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
                >
                  <img
                    alt="Black"
                    className="StylesComponent__SelectedImage-ajk3ts-6 eDgpFT"
                    id={70550}
                    onClick={[Function]}
                    src="https://images.unsplash.com/photo-1552902865-b72c031ac5ea?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
                  />
                </StylesComponent__SelectedImage>
                <StylesComponent__Image
                  alt="Grey"
                  id={70551}
                  key="70551"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  src="https://images.unsplash.com/photo-1562542082-519ebcdb43e6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
                >
                  <img
                    alt="Grey"
                    className="StylesComponent__Image-ajk3ts-5 cdIpah"
                    id={70551}
                    onClick={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    src="https://images.unsplash.com/photo-1562542082-519ebcdb43e6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
                  />
                </StylesComponent__Image>
                <StylesComponent__Image
                  alt="Goldenrod"
                  id={70552}
                  key="70552"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  src="https://images.unsplash.com/photo-1530073391204-7b34a1497281?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
                >
                  <img
                    alt="Goldenrod"
                    className="StylesComponent__Image-ajk3ts-5 cdIpah"
                    id={70552}
                    onClick={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    src="https://images.unsplash.com/photo-1530073391204-7b34a1497281?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
                  />
                </StylesComponent__Image>
                <StylesComponent__Image
                  alt="Maroon"
                  id={70553}
                  key="70553"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  src="https://images.unsplash.com/photo-1519857609704-61e751edba25?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
                >
                  <img
                    alt="Maroon"
                    className="StylesComponent__Image-ajk3ts-5 cdIpah"
                    id={70553}
                    onClick={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    src="https://images.unsplash.com/photo-1519857609704-61e751edba25?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
                  />
                </StylesComponent__Image>
                <StylesComponent__Image
                  alt="Chartreuse"
                  id={70554}
                  key="70554"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  src="https://images.unsplash.com/photo-1530073391204-7b34a1497281?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
                >
                  <img
                    alt="Chartreuse"
                    className="StylesComponent__Image-ajk3ts-5 cdIpah"
                    id={70554}
                    onClick={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    src="https://images.unsplash.com/photo-1530073391204-7b34a1497281?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
                  />
                </StylesComponent__Image>
                <StylesComponent__Image
                  alt="White"
                  id={70555}
                  key="70555"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  src="https://images.unsplash.com/photo-1510390099355-23e690d8129d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
                >
                  <img
                    alt="White"
                    className="StylesComponent__Image-ajk3ts-5 cdIpah"
                    id={70555}
                    onClick={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    src="https://images.unsplash.com/photo-1510390099355-23e690d8129d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
                  />
                </StylesComponent__Image>
              </div>
            </StylesComponent__Thumbs>
          </div>
        </StylesComponent__StyleWrapper>
      </Styles>
      <ProductOverview__Purchasing>
        <div
          className="ProductOverview__Purchasing-sc-1yvw4zm-6 hUHZdp"
        >
          <Selectors
            skus={
              Object {
                "407566": Object {
                  "quantity": 8,
                  "size": "XS",
                },
                "407567": Object {
                  "quantity": 16,
                  "size": "S",
                },
                "407568": Object {
                  "quantity": 17,
                  "size": "M",
                },
                "407569": Object {
                  "quantity": 10,
                  "size": "L",
                },
                "407570": Object {
                  "quantity": 15,
                  "size": "XL",
                },
                "407571": Object {
                  "quantity": 6,
                  "size": "XXL",
                },
              }
            }
          >
            <Selectors__Wrapper>
              <div
                className="Selectors__Wrapper-sc-1txd6fn-0 hnvKpv"
              >
                <SizeDropdown
                  list={
                    Array [
                      Object {
                        "quantity": 8,
                        "size": "XS",
                      },
                      Object {
                        "quantity": 16,
                        "size": "S",
                      },
                      Object {
                        "quantity": 17,
                        "size": "M",
                      },
                      Object {
                        "quantity": 10,
                        "size": "L",
                      },
                      Object {
                        "quantity": 15,
                        "size": "XL",
                      },
                      Object {
                        "quantity": 6,
                        "size": "XXL",
                      },
                    ]
                  }
                  resetThenSet={[Function]}
                  title="SELECT SIZE"
                >
                  <SizeDropdown__DDWrapper>
                    <div
                      className="SizeDropdown__DDWrapper-u3ohyw-0 knauff"
                    >
                      <SizeDropdown__DDHeader
                        onClick={[Function]}
                      >
                        <button
                          className="SizeDropdown__DDHeader-u3ohyw-1 dFNLhV"
                          onClick={[Function]}
                        >
                          <SizeDropdown__DDHeaderTitle>
                            <div
                              className="SizeDropdown__DDHeaderTitle-u3ohyw-2 fzssNQ"
                            >
                              SELECT SIZE
                            </div>
                          </SizeDropdown__DDHeaderTitle>
                          <FontAwesome
                            name="angle-down"
                            size="2x"
                          >
                            <span
                              aria-hidden={true}
                              className="fa fa-angle-down fa-2x"
                            />
                          </FontAwesome>
                        </button>
                      </SizeDropdown__DDHeader>
                    </div>
                  </SizeDropdown__DDWrapper>
                </SizeDropdown>
                <QuantityDropdown
                  available={null}
                  list={
                    Array [
                      1,
                      2,
                      3,
                      4,
                      5,
                      6,
                      7,
                      8,
                      9,
                      10,
                      11,
                      12,
                      13,
                      14,
                      15,
                    ]
                  }
                  resetThenSet={[Function]}
                  title="–"
                >
                  <QuantityDropdown__DDWrapper>
                    <div
                      className="QuantityDropdown__DDWrapper-sc-1fwgee9-0 kbjxSC"
                    >
                      <QuantityDropdown__DDHeader
                        onClick={[Function]}
                      >
                        <button
                          className="QuantityDropdown__DDHeader-sc-1fwgee9-1 liwBJT"
                          onClick={[Function]}
                        >
                          <QuantityDropdown__DDHeaderTitle>
                            <div
                              className="QuantityDropdown__DDHeaderTitle-sc-1fwgee9-2 kBnByu"
                            >
                              –
                            </div>
                          </QuantityDropdown__DDHeaderTitle>
                          <FontAwesome
                            name="angle-down"
                            size="2x"
                          >
                            <span
                              aria-hidden={true}
                              className="fa fa-angle-down fa-2x"
                            />
                          </FontAwesome>
                        </button>
                      </QuantityDropdown__DDHeader>
                    </div>
                  </QuantityDropdown__DDWrapper>
                </QuantityDropdown>
              </div>
            </Selectors__Wrapper>
          </Selectors>
          <ProductOverview__PurchaseLikeButtons>
            <div
              className="ProductOverview__PurchaseLikeButtons-sc-1yvw4zm-7 dHNLnu"
            >
              <ProductOverview__AddToCart>
                <button
                  className="ProductOverview__AddToCart-sc-1yvw4zm-8 coXpub"
                >
                  <div>
                    ADD TO BAG
                  </div>
                  <ProductOverview__Plus>
                    <div
                      className="ProductOverview__Plus-sc-1yvw4zm-9 fEDTqw"
                    >
                      +
                    </div>
                  </ProductOverview__Plus>
                </button>
              </ProductOverview__AddToCart>
              <ProductOverview__Like>
                <button
                  className="ProductOverview__Like-sc-1yvw4zm-10 bKuQWw"
                >
                  ☆
                </button>
              </ProductOverview__Like>
            </div>
          </ProductOverview__PurchaseLikeButtons>
        </div>
      </ProductOverview__Purchasing>
    </div>
  </ProductOverview__Wrapper>
</ProductOverview>
`;
